src/SignhostAPIClient/Rest/DataObjects/Signer.cs

```diff
@@ -37,6 +37,7 @@ public class Signer
 
 		public bool? SendSignConfirmation { get; set; }
 
+		[Obsolete("Use transaction.DaysToRemind instead")]
```

Comment:
I guess this should be signer[].DaysToRemind?
Maybe also mention that this is a No-Op property?